README.md

```diff
@@ -291,12 +291,28 @@ Key:
 
 </details>
 <details>
-<summary>MS MARCO V2.1 Document Regressions</summary>
+<summary>MS MARCO V2.1 Passage Regressions (TREC RAG 24)</summary>
```

Comment:
Terminology here can be misleading, if we are doing segmented maybe we should say that instead of passage?